tests/apps/madeline/test_madeline.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for CGs madeline app"""
+from cg.apps.madeline import make_ped, run
+
+
+def test_run_madeline(madeline_process, madeline_input):
+    # GIVEN a ped stream and a madeline process mock
+    # WHEN running the madeline command
```

Comment:
suggest having an empty line before each # GIVEN, # WHEN and # THEN to make them a bit easier to read